sonarqube/pyproject.toml

```diff
@@ -9,7 +9,6 @@ build-backend = "hatchling.build"
 name = "datadog-sonarqube"
```

Comment:
Not completely related to your PR, although I saw in the `classifiers` section that it only has 
```
"Programming Language :: Python :: 3.8",
``` 
included, although it looks like [py27 is supported](https://github.com/DataDog/integrations-core/blob/a2990efc7c8e900c1578acc609e186d9cdf07614/sonarqube/hatch.toml#L4). Should `Python :: 2.7` be included?